components/header_footer_route/content_layouts/column.tsx

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import Constants from 'utils/constants';
+
+import './column.scss';
+
+type ColumnProps = {
+    title: string;
+    message: string;
+    SVGElement?: React.ReactNode;
+    extraContent?: React.ReactNode;
+    onEnterKeyDown?: (e: React.KeyboardEvent<HTMLInputElement>) => void;
+}
+
+const Column = ({title, message, SVGElement, extraContent, onEnterKeyDown}: ColumnProps) => {
+    const handleOnEnterKeyDown = (e: React.KeyboardEvent<HTMLInputElement>) => {
+        if (onEnterKeyDown && e.key === Constants.KeyCodes.ENTER[0]) {
```

Comment:
We could use `Utils.isKeyPressed` for better compatibility

https://github.com/mattermost/mattermost-webapp/blob/c06a4520f9589df73bd3062028ef106d4b1bc31b/utils/utils.tsx#L112